ProductionSolenoidGeom/src/PSEnclosure.cc

```diff
@@ -28,7 +28,7 @@ namespace mu2e {
 
       }
 
-    } else {
+    } else if(pse.version() == 2){
```

Comment:
What happens if pse.version() is neither 1 nor 2?   Please add an "else" clause at the end, e.g. throw an exception with a message about lacking support for other enclosure versions.

Reply:
I've added a print option for the new version of the PS enclosure, version 3.